sympy/sets/tests/test_fancysets.py

```diff
@@ -518,7 +518,9 @@ def test_ComplexRegion_contains():
     r1 = Interval(0, 1)
     theta1 = Interval(0, 2*S.Pi)
     c3 = ComplexRegion(r1*theta1, polar=True)
-    assert 0.5 + 0.6*I in c3
+    #assert (0.5 + 6*I/10) in c3
+    assert (S.Half + .6*I) in c3
+    assert (S.Half + 6*I/10) in c3
```

Comment:
What's happening here?

Reply:
Hmmm. I will have to check. I would expect 0.5 to have passed and 0.6 to have failed. Given the two that are passing I would expect the comment-out one to pass, too.